cmake/modules/AddCephTest.cmake

```diff
@@ -1,5 +1,22 @@
 #AddCephTest is a module for adding tests to the "make check" target which runs CTest
 
+execute_process(
+    COMMAND uname -m
+    OUTPUT_VARIABLE ARCHSTR
+    OUTPUT_STRIP_TRAILING_WHITESPACE)
+
+if(LINUX)
+  set(OSSTR linux)
+  set(PYVER 2.7)
```

Comment:
@tchaikov 
I'm all in favour for as much flexibility as possible.
Especially with with Python going to 3.x, this would prevent more trouble.
I looked at the trace-expand output of Cmake to see if I could find anything you suggest, but did not see any.
But I'll give your suggestion a run.